test/parallel/test-fs-timestamp-parsing-error.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+require('../common');
+var assert = require('assert');
+var fs = require('fs');
+
+[undefined, null, {}, []].forEach((input) => throws(input));
+
+function throws(input) {
+  assert.throws(() => fs._toUnixTimestamp(input)
+  , 'Error: Cannot parse time: ' + input);
```

Comment:
The second argument here should be either the type of error thrown or a check function. You can use `common.expectsError()` (see [here](https://github.com/nodejs/node/tree/master/test#expectserrorsettings))